controller: "hj_controller"
goal:
  coordinates:
    - 1.5
    - 1.5
    - 1.57
  padding:
    - .1
    - .1
    - 3.14

grid_domain:
  lo:
    - -6.0
    - -6.0
    - 0.0
  hi:
    - 6.0
    - 6.0
    - 6.29
  resolution:
    - 51
    - 51
    - 51

control_bounds:
  lo:
    - -1.0
    - 0.0
  hi:
    - 1.0
    - 0.2

time:
  time_index: 80
  time_horizon: -10
  rate: 4


